src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1890,11 +1890,11 @@ private async Task DeleteResourceGroupIfUserConsentsAsync()
             }
         }
 
-        private async Task<bool> RunTestWorkflow(IStorageAccount storageAccount, bool usePreemptibleVm = true)
+        private async Task<bool> RunTestWorkflow(IStorageAccount storageAccount, bool usePreemptibleVm = true, string UbuntuImageName )
```

Comment:
Should be `private async Task<bool> RunTestWorkflow(IStorageAccount storageAccount, bool usePreemptibleVm = true, string UbuntuImageName = default)`

Reply:
instead of `UbuntuImageName`, wouldn't "testWdlDockerImageName" be more appropriate?  Remember that in C# method parameters, variable names are lowercase to start